dashboard/app/helpers/openai_chat_helper.rb

```diff
@@ -28,6 +32,41 @@ def self.request_chat_completion(messages)
     )
   end
 
+  # Used to check safety content given text with the given moderation system prompt.
+  def self.request_safety_check(text, safety_system_prompt)
+    # Set up the API endpoint URL and request headers
+    headers = {
+      "Content-Type" => "application/json",
+      "Authorization" => "Bearer #{OPENAI_AICHAT_SAFETY_API_KEY}"
+    }
+    # Format messages with text to be checked for safety and moderation system prompt.
+    messages = [
+      {
+        role: "system",
+        content: safety_system_prompt
+      },
+      {
+        role: "user",
+        content: text
+      }
+    ]
+    data = {
+      model: AICHAT_SAFETY_GPT_MODEL,
+      temperature: AICHAT_SAFETY_TEMPERATURE,
```

Comment:
This should be one to match our testing behavior, but I'd suggest not hardcoding a temperature and leaving this param blank (its not required, and this allows openai to optimize for what they think the best default temperature is).

Reply:
Oh interesting - thanks!
So just confirming that during your testing you left this param blank?

Reply:
Confirmed, I used the default value for this param (which I believe results in a temperature of 1, but I would suggest skipping the param to match test results as closely as possible)

Reply:
Link to comment about this temperature setting: https://docs.google.com/document/d/1LB7Pc-X5RU52dUTqN4M8cruDrXodR85Xvo05aBdwAus/edit?disco=AAABWlU2K8Q